Reactions/Fuego/GPU/arkode/reactor.cpp

```diff
@@ -51,10 +51,10 @@ int react(const amrex::Box& box,
     realtype time_init, time_out;
 
     void *arkode_mem    = NULL;
-    N_Vector y         = NULL;
+    N_Vector y          = NULL;
 
     NEQ            = NUM_SPECIES+1;
-    NCELLS         = Ncells;
+    NCELLS         = box.numPts();
```

Comment:
Is there a potential issue for large box blowing this out (should it be an unsigned long)?

Reply:
we should probably be safe unless someone is using max_grid_size > 1024. I can change it out to unsigned just to make sure.

Reply:
Maybe a better option is to leave it as int for compatibility with SUNDIALS, but to the test in an ALWAYS_ASSERT to make sure it's safe

Reply:
I have added an AMREX_ASSERT in there now